environments/custom/files/testbed_ceph_devices.fact

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python3
+
+import json
+import os
+from subprocess import PIPE, run
+
+import requests
+
+response = requests.get('http://169.254.169.254/latest/meta-data/block-device-mapping/root')
+root = os.path.basename(response.text)
+
+devices = run("lsblk -nd -I 8 --output NAME | sort | grep -v %s" % root, stdout=PIPE, stderr=PIPE, universal_newlines=True, shell=True)
```

Comment:
Don't limit to SCSI (-I 8). You need to allow sda (8), vda (252), xvda (220) at least: -I 8,220,252.